XX.md

```diff
@@ -0,0 +1,266 @@
+NIP-XX
+======
+
+Marketplace Protocol
+---------------------------
+
+`draft` `optional`
+
+This NIP defines a comprehensive protocol for implementing decentralized marketplaces on Nostr, combining and enhancing the approaches from [NIP-15](15.md) and [NIP-99](99.md). It provides a complete e-commerce framework while maintaining the protocol's simplicity and interoperability.
+
+The focus for NIP-99 is on the product listing but this NIP also provides a structure for other necessary flows for a complete checkout procedure:
+
+1) Order Communication Flow 
+2) Shipping Calculation Flow
+   This will be it's own kind ( replaceable event ) for updates during the shipping process, pricing schema for shipping costs and general shipping settings: Examples are zoning restrictions: USA, EU, NA, and free shipping: 50 USD
+3) Payment Flows
+
+A general checkout implementation can be structured as followed:
+1) Items are gathered in a Local Basket 
+2) Shipping details are required to calculate a full payment amount
+3) Order gets send once Payment is received ( Zap Receipt , Nut Receipt, Marketplace server )
+
+In practice a shop can have a preferred checkout and shipping option defined at the store level and product level. A option at the store level will be used if nothing is defined at the product level, essentially making it that product level shipping and payment options always overrule store level settings.
+
+## Events and Kinds
+
+### Product Listing (Kind: 30402)
+Following NIP-99's schema for product representation:
+
+```jsonc
+{
+  "kind": 30402,
+  "created_at": <unix timestamp>,
+  "content": "<product description in markdown>",
+  "tags": [
+    ["d", "<product identifier>"],
+    ["title", "<product title>"],
+    ["price", "<amount>", "<currency>", "<optional frequency>"],
+    // Optional tags
+    ["shipping", "<shipping-option-id>"], // References to shipping options
+    ["summary", "<short description>"],
+    ["image", "<url>", "<dimensions>"],
+    ["location", "<location string>"],
+    ["g", "<geo hash>"],
+    ["t", "<category>"],
+    ["a", "<product-list-id>"] // Reference to product collection if applicable
+  ]
+}
+```
+
+### Product Draft (Kind: 30403)
```

Comment:
No need for a new kind here you can just use: https://github.com/nostr-protocol/nips/blob/master/37.md